scripts/package

```diff
@@ -6,7 +6,7 @@ source $(dirname $0)/version
 cd $(dirname $0)/..
 
 mkdir -p dist/artifacts
-cp bin/backup-restore-operator dist/artifacts/backup-restore-operator${SUFFIX}
+cp bin/backup-restore-operator* dist/artifacts/
```

Comment:
I believe the intent of this was to copy the compiled binary into the artifacts folder. However I'm unsure if it should be a single binary as an artifact that matches the chart it generates? Or if it should be all of the binary artifacts of this build run - meaning if `CROSS=true` we should actually be copying all the bins for all the arch/platforms?